src/components/Sidebar.jsx

```
@@ -0,0 +1,60 @@
+import React from "react";
+import { Link, useLocation } from "react-router-dom";
+
+const Sidebar = () => {
+  const location = useLocation();
+  const isBeranda = location.pathname === "/DashboardBeranda";
+  const isIkutLomba = location.pathname === "/IkutLomba";
+
+  const sharedSpanStyle = {
```

ESLint

[error] 'sharedSpanStyle' is assigned a value but never used. Allowed unused vars must match /^[A-Z_]/u.

(no-unused-vars)